twisted/web/_http2.py

```diff
@@ -625,7 +625,8 @@ def _handleWindowUpdate(self, event):
                 # unnecessary but benign. We'll ignore it.
                 return
 
-            self.priority.unblock(streamID)
+            if self._outboundStreamQueues.get(streamID):
```

Comment:
Thanks! Maybe it is worth leaving a comment for future generations which might want to refactor this based on EAFP